src/components/interfaces.tsx

```diff
@@ -0,0 +1,18 @@
+export interface Content {
```

Comment:
So I like the idea of an external interfaces file but only if you think we're going to need to use these across multiple components (which I think we will so it's fine).

One thing to note that if it were up to me, I would make a folder called `types` with a file inside called `index.ts`. When you do this you can still import `"../types"` without needing to append index.tsx to the end of the filepath. Note that we don't need to make this file `.tsx` anymore since we don't have any JSX inside of it.